src/currency.ts

```diff
@@ -0,0 +1,14 @@
+export function SpaceCurrency(initialCurrency:number) {
```

Comment:
Don't write function in such way. Functions' name should be camelCase. By the way you can choose better name, something like `generateCurrency`, and plz, use arrow functions. Arrow functions look pretty good, but current is nice too, Also don't forget about return type of your function.

**Example:**
`function generateCurrency(n: number): string {};`